### ISSUE TITLE
Upgrade all option?

### ISSUE THREAD

**Issue by ronnietucker** [UbuntuOpenStore/openstore-meta#165]
Might be worthwhile adding an 'upgrade/update all' button. Rather than having the user update/upgrade one by one. I just did it for 5 apps and it was a bit of a nuisance.

MX4 and 16.04/dev

**Comment by pixelr0** [comment]
yeah, i had to update 20 apps today. not fun to update em one by one :D
